markdown/developers/developer_tutorials.md

```diff
@@ -7,7 +7,7 @@ subtitle: In-depth step-by-step guides for writing nf-core pipelines
 
 This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core.
 
-- [Tutorial: Create a DSL2 Module](tutorials/dsl2_modules_tutorial)
+- [Tutorial: Create a DSL2 Module](tutorials/dsl2_modules_tutorial.md)
```

Comment:
The `.md` suffix _shouldn't_ be needed here I think.. Don't the rewriting rules chop it off?

Reply:
It isn't needed for the website rendering. But the broken link checker looks at the markdown and then checks https://nf-co.re/tutorials/dsl2_modules_tutorial in this case, which doesn't resolve. Adding the file extensions makes the test pass (see link to tutorial for unofficial pipelines).